configure.ac

```diff
@@ -152,6 +152,16 @@ if test x"$libiscsi_cv_HAVE_LINUX_ISER" = x"yes"; then
 fi
 AM_CONDITIONAL([HAVE_LINUX_ISER], [test $libiscsi_cv_HAVE_LINUX_ISER = yes])
 
+AC_CACHE_CHECK([for RDMA ack timeout support],libiscsi_cv_HAVE_RDMA_ACK_TIMEOUT,[
+AC_TRY_COMPILE([
+#include <rdma/rdma_cma.h>],
+[int option = RDMA_OPTION_ID_ACK_TIMEOUT;],
```

Comment:
Can this trigger a compiler warning about the 'option' variable not being used? How about changing this into "return RDMA_OPTION_ID_ACK_TIMEOUT;"?

Reply:
OK